libs/features/mas/src/aem-fragment.js

```diff
@@ -224,7 +224,7 @@ export class AemFragment extends HTMLElement {
     }
 
     #transformPublishData() {
-        const { fields, id, tags,  } = this.#rawData;
+        const { fields, id, tags, } = this.#rawData;
```

Comment:
comma is not needed when on the same line

Reply:
sure